src/audit.js

```diff
@@ -52,7 +52,7 @@ let playAudit = async function (auditConfig = {}) {
     ...auditConfig.reports,
   };
 
-  const { errors, results } = await lighthouse({
+  const { comparison, results } = await lighthouse({
```

Comment:
just curious, is it a breaking change with compared to the earlier versions and dependent workflows?

Reply:
I think it doesn't break previous implementations. I only changed the way "lighthouse" returns information.

Previously it returned the result of the `compare(thresholds, newValues)` function, now that is sotred in the `comparison` object and results become the report object.

I tested it for positive and negative use cases (with and without errors) and it works nice. (after the last patch hehe)